### PR TITLE
fix: Colorize example tests

### PR DESCRIPTION
**Pull request by karlkfi**
Makes the `expectedOutputLine` and `expectedNotFound` tests succeed green and fail red, with re-printed expectation on failure, to make it easier to see problems at a glance when running locally.

### CODE REVIEW

**Comment by karlkfi** [comment]
/test cli-utils-presubmit-master-e2e

flaky failure?

**Comment by karlkfi** [comment]
Tests fixed.

File: examples/alphaTestExamples/MultipleServices.md
Comment (karlkfi) [reply examples/alphaTestExamples/MultipleServices.md:56]: Go tests have the concept of Error vs Fatal, where error continues and fatal does not. I'm not sure which tests in the examples should be errors and which should be fatal, so I assumed they were all fatal. Do you know the tests well enough to make the distinction for each one?

File: examples/alphaTestExamples/MultipleServices.md
Comment (karlkfi) [reply examples/alphaTestExamples/MultipleServices.md:58]: If you have an idea of how to do that succinctly, I'm all ears. I decided not to re-print the entire output again. And I don't know how to easily get the command run or the line number... Got ideas?

Comment (karlkfi) [reply examples/alphaTestExamples/MultipleServices.md:58]: FWIW, with every test fatal, we can at least tell which code block caused the failure, if not the exact line.

File: examples/alphaTestExamples/MultipleServices.md
Comment (mortent) [reply examples/alphaTestExamples/MultipleServices.md:56]: I think failing on the first error within each md file probably make sense. The commands depend on each other, so continuing will likely just result in other errors.
Can we just run mdrip once for each file and continue even if one of them fails?

File: examples/alphaTestExamples/MultipleServices.md
Comment (mortent) [reply examples/alphaTestExamples/MultipleServices.md:58]: I think the only reasonable way to do this would be to print the text that was expected. But it might be that it clutters  up the output so it isn't much help.

File: examples/alphaTestExamples/MultipleServices.md
Comment (mortent) [reply examples/alphaTestExamples/MultipleServices.md:56]: This is something we can change quickly at any time if we change our minds, so I'm not super worried about it either way.

Comment (karlkfi) [reply examples/alphaTestExamples/MultipleServices.md:56]: > Can we just run mdrip once for each file and continue even if one of them fails?

Done.

**Comment by karlkfi** [comment]
Rebased and modified the test script to execute each file individually, so that the new exit on error behavior only affects the file being tested.

Please re-review.

**Comment by karlkfi** [comment]
`pruneAndDelete.md` was using the `expectedNotFound` function from `helloapp.md`, causing test failure now that they exec in different mdrip contexts (I didn't know shell variables were shared between files).

Fixed by copying `expectedNotFound` to `pruneAndDelete.md`.

**Comment by karlkfi** [comment]
Tests passed.

**Review by seans3 — APPROVED** [review]
I'll let others have look before merge.

**Review by mortent — COMMENTED** [review]
/lgtm

**Pull request merged**
